### ISSUE TITLE
Abnormal circulating Osteopontin concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6111]
**Preferred term label:** Abnormal circulating Osteopontin concentration

**Synonyms** Abnormal circulating OPN.

**Definition (free text, please give PubMed ID)** Concentration of osteopontin in the blood is outside the normal range., PMID: 30003880

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0033123
